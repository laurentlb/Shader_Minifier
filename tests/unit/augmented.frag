int foo(int x, int y) {
    int a=x;
    a=a+x;
    a=a*y;
    a=a>>x;
    a=a^y;
    a=a|y;
    return a;
}

void main() {
    foo(0, 0);
}
