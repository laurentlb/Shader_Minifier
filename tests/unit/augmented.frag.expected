int foo(int x,int y)
{
  int a=x;
  a+=x;
  a*=y;
  a>>=x;
  a^=y;
  a|=y;
  return a;
}
void main()
{
  foo(0,0);
}
